packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -116,6 +133,9 @@ contract Attestations is IAttestations, Ownable, Initializable, UsingRegistry, R
   mapping(bytes32 => IdentifierState) identifiers;
   mapping(address => Account) accounts;
 
+  // Mapping of attestationKey addresses to account addresses
+  mapping(address => address) keysToAccount;
```

Comment:
`accountFromAttestationKey`?

Reply:
Renamed it to `authorizedAttestors` in line with your comment below, wdyt?

Reply:
That name makes me think the mapping will be in the other direction. `authorizedBy`?